token/services/db/sql/common/tokens.go

```diff
@@ -1149,13 +1157,15 @@ func (t *TokenTransaction) Delete(ctx context.Context, tokenID token.ID, deleted
 	// logger.Debugf("delete token [%s:%d:%s]", txID, index, deletedBy)
 	// We don't delete audit tokens, and we keep the 'ownership' relation.
 	now := time.Now().UTC()
-	query, err := NewUpdate(t.table.Tokens).Set("is_deleted, spent_by, spent_at").Where("tx_id, idx").Compile()
-	if err != nil {
-		return errors.Wrapf(err, "failed building query")
-	}
+	query, args := q.Update(t.table.Tokens).
+		Set("is_deleted", true).
+		Set("spent_by", deletedBy).
+		Set("spent_at", now).
```

Comment:
`now` seems to be used only once. I would inline it. same for that one occurrence above and one below